invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/index.js

```diff
@@ -120,20 +122,36 @@ class DashboardMenu extends Component {
             </Menu>
           </Container>
         </Container>
-        <Container>
-          <Segment className="borderless shadowless p-0">
-            <div className="rel-pt-2">
-              <SearchApp
-                appName={activeContent.appId}
-                key={activeContent.appId}
-                config={activeContent.config}
-              />
-            </div>
-          </Segment>
-        </Container>
+        {!menuOnly && (
```

Comment:
The `menuOnly` is to keep the 3 tabs (Records, Communities, Requests) and show the request details inside?
You mentioned that another solution would be to have the menu in the Jinja template and this would be avoided... feasible in a short time?